spark_llm/prompt.py

```diff
@@ -119,3 +119,64 @@
 Assume the result of the Spark SQL query is stored in a dataframe named 'df', visualize the query result using plotly.
 There is no need to install any package with pip.
 """
+
+VERIFY_TEMPLATE = """
+Given 1) a PySpark dataframe, my_df, and 2) a description of expected properties, desc,
```

Comment:
Shall we use `df` instead of `my_df`? Also, it would be great to quote the variable name